garden_ai/app/model.py

```diff
@@ -71,6 +71,11 @@ def register(
             f"Sorry, we only support 'sklearn', 'tensorflow', and 'pytorch'. The {flavor} flavor is not yet supported."
         )
 
+    if name == "YOUR MODEL's NAME HERE":
+        raise typer.BadParameter(
+            "Model name 'YOUR MODEL's NAME HERE' is reserved for scaffolded pipeline example. Please change your model name to something else."
+        )
```

Comment:
I'm not sure we need this check. I think we would already throw an error at Pydantic validation for `LocalModel` because model names aren't allowed to have spaces. (Or pretty much anything that isn't alphanumeric.) Also, while it's easy for someone to use the name in the scaffolded pipeline by mistake, someone would have to be deliberate to upload a model with this name. I'd lean towards taking out this check assuming that other validation would catch it, but open to other takes.